src/library/Icon/Icon.js

```diff
@@ -8,7 +8,7 @@ type Props = {
   size?: 'small' | 'medium' | 'large' | string,
```

Comment:
Is this actually a Breaking Change? Both the prop & the theme APIs are the same. The only change is the default fill value.

Reply:
I think so, because it's effectively removing the default value, which could have quite noticeable consequences in a consuming app.